### PR TITLE
Add `Disambiguation/Score`; make individual articles of some `Score` sections

### PR DESCRIPTION
**Pull request by TicClick**
part of #6431

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

cc @DoryanR for proof-reading French versions

### CODE REVIEW

File: wiki/Score/Total_score/fr.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Score total
+
+*Pour d'autres utilisations, voir : [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
+
+Le score total est la **somme de tous les scores sur des beatmaps classées et/ou approuvées, réussis ou non.**
```
Comment (DoryanR) [inline wiki/Score/Total_score/fr.md:5]:
```suggestion
Le score total est la **somme de tout les scores sur des beatmaps classées et/ou approuvées, réussis ou non.**
```

File: wiki/Score/Total_score/fr.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Si un joueur joue une beatmap classée pour la première fois et qu'il obtient 500k, les deux scores augmentent de 500k.
+Ensuite, si le joueur rejoue la beatmap et qu'il obtient un score de 400k, seul le score total augmentera de 400k.
+
+## Niveau
+
+Le niveau d'un joueur est basé uniquement sur le **score total** du joueur. Le score requis pour passer un niveau est donné par la fonction suivante :
```
Comment (DoryanR) [inline wiki/Score/Total_score/fr.md:12]:
```suggestion
Le niveau d'un joueur est basé uniquement sur le **score total** du joueur. Le score requis pour passer un niveau est donné par la formule suivante : 
```

File: wiki/Score/Total_score/fr.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## Niveau
+
+Le niveau d'un joueur est basé uniquement sur le **score total** du joueur. Le score requis pour passer un niveau est donné par la fonction suivante : 
+
+```
+score(n) = 5,000 / 3 * (4n^3 - 3n^2 - n) + 1.25 * 1.8^(n - 60) si n <= 100
+score(n) = 26,931,190,827 + 99,999,999,999 * (n - 100)         si n > 100
+```
+
+Notez qu'à cause de la technique d'arrondi utilisée par le jeu, les valeurs pour les niveaux supérieurs à 100 ne correspondent pas exactement à la formule. Cela est dû au fait que le jeu conserve une liste pré-calculée des différences de score entre chaque niveau, et que chaque élément de cette liste est arrondi à l'entier le plus proche. Par exemple, le 80e élément de la liste est `score(81) - score(80) = 128,927,482.36216`, arrondi à 128,927,482. Au niveau 100, l'erreur totale est de +1.629, ce qui explique pourquoi `score(100) = 26,931,190,828.629` ne correspond pas au 26,931,190,827 dans la deuxième partie de la fonction.
```
Comment (DoryanR) [inline wiki/Score/Total_score/fr.md:19]:
```suggestion
Notez qu'à cause des arrondis par le jeu, les valeurs pour les niveaux supérieurs à 100 ne correspondent pas exactement à la formule. Cela est dû au fait que le jeu conserve une liste précalculée des différences de score entre chaque niveau, et que chaque élément de cette liste est arrondi à l'entier le plus proche. Par exemple, le 80e élément de la liste est `score(81) - score(80) = 128,927,482.36216`, arrondi à 128,927,482. Au niveau 100, l'erreur totale est de +1.629, ce qui explique pourquoi `score(100) = 26,931,190,828.629` ne correspond pas à 26,931,190,827 dans la deuxième partie de la formule.
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
Nothing to say except that for the French side, I appreciate the initiative.

File: wiki/Score/Ranked_score/fr.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Score classé
+
+*Pour d'autres utilisations, voir : [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
```
Comment (Joehuu) [inline wiki/Score/Ranked_score/fr.md:7]:
```suggestion
*Pour d'autres utilisations, voir [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
```

File: wiki/Score/Total_score/en.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Total score
+
+*For other uses, see [Score (disambiguation)](/wiki/Disambiguation/Score).*
+
+The total score is the **total sum of all scores on all beatmaps, including failed attempts.**
+
+If a player plays a ranked song for the first time and gets 500k, both ranked and total score increase by 500k.
+Then, the player plays again and gets 400k, only the total score increases by 400k.
```
Comment (Joehuu) [inline wiki/Score/Total_score/en.md:8]:
```suggestion
If a player plays a ranked song for the first time and gets 500k, both ranked and total score increase by 500k. Then, the player plays again and gets 400k, only the total score increases by 400k.
```

File: wiki/Score/Total_score/fr.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Score total
+
+*Pour d'autres utilisations, voir : [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
```
Comment (Joehuu) [inline wiki/Score/Total_score/fr.md:3]:
```suggestion
*Pour d'autres utilisations, voir [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
```

File: wiki/Score/Total_score/fr.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+Le score total est la **somme de tout les scores sur des beatmaps classées et/ou approuvées, réussis ou non.**
+
+Si un joueur joue une beatmap classée pour la première fois et qu'il obtient 500k, les deux scores augmentent de 500k.
+Ensuite, si le joueur rejoue la beatmap et qu'il obtient un score de 400k, seul le score total augmentera de 400k.
```
Comment (Joehuu) [inline wiki/Score/Total_score/fr.md:8]:
```suggestion
Si un joueur joue une beatmap classée pour la première fois et qu'il obtient 500k, les deux scores augmentent de 500k. Ensuite, si le joueur rejoue la beatmap et qu'il obtient un score de 400k, seul le score total augmentera de 400k.
```

File: wiki/Score/fr.md [hunk 8]
```diff
@@ -6,6 +6,8 @@ outdated_since: c78e8f94260067c49d36a55deaaf7f40cb796b39
 
 # Score
 
+*Pour d'autres utilisations, voir : [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
```
Comment (Joehuu) [inline wiki/Score/fr.md:9]:
```suggestion
*Pour d'autres utilisations, voir [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
```

**Review by Joehuu — CHANGES_REQUESTED** [review]

**Review by DoryanR — APPROVED** [review]
Good for the French side

File: wiki/History_of_osu!/Online_rankings/osu!/fr.md [hunk 9]
```diff
@@ -21,7 +21,7 @@ Voici la liste des joueurs reconnus comme ayant atteint le rang 1 dans le [class
 
 Au fil des ans, trois systèmes différents ont été utilisés pour classer les joueurs :
 
-- [Ranked Score](/wiki/Score#ranked-score) (6 octobre 2007 - 24 juillet 2012)
+- [Ranked Score](/wiki/Score/ranked_score) (6 octobre 2007 - 24 juillet 2012)
```
Comment (TicClick) [inline wiki/History_of_osu!/Online_rankings/osu!/fr.md:24]:
```suggestion
- [Ranked Score](/wiki/Score/Ranked_score) (6 octobre 2007 - 24 juillet 2012)
```

**Review by Joehuu — DISMISSED** [review]

File: wiki/Score/Ranked_score/en.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Ranked score
+
+*For other uses, see [Score (disambiguation)](/wiki/Disambiguation/Score).*
+
+The ranked score is the **total sum of the best scores for all ranked difficulties played online.**
```
Comment (TPGPL) [inline wiki/Score/Ranked_score/en.md:9]:
```suggestion
The **ranked score** is the total sum of the best scores for all ranked difficulties played online.
```

File: wiki/Score/Ranked_score/fr.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Score classé
+
+*Pour d'autres utilisations, voir [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
+
+Le score classé est la **somme des meilleurs scores sur toutes les beatmaps classées jouées en ligne.**
```
Comment (TPGPL) [inline wiki/Score/Ranked_score/fr.md:9]:
```suggestion
Le **score classé** est la somme des meilleurs scores sur toutes les beatmaps classées jouées en ligne.
```

File: wiki/Score/Total_score/en.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
+# Total score
+
+*For other uses, see [Score (disambiguation)](/wiki/Disambiguation/Score).*
+
+The total score is the **total sum of all scores on all beatmaps, including failed attempts.**
```
Comment (TPGPL) [inline wiki/Score/Total_score/en.md:5]:
```suggestion
The **total score** is the total sum of all scores on all beatmaps, including failed attempts.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Score/Total_score/fr.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+# Score total
+
+*Pour d'autres utilisations, voir [Score (désambiguïsation)](/wiki/Disambiguation/Score).*
+
+Le score total est la **somme de tout les scores sur des beatmaps classées et/ou approuvées, réussis ou non.**
```
Comment (Joehuu) [inline wiki/Score/Total_score/fr.md:5]: Missed this.
```suggestion
Le **score total** est la somme de tout les scores sur des beatmaps classées et/ou approuvées, réussis ou non.
```

Comment (TicClick) [reply wiki/Score/Total_score/fr.md:5]: ah, thanks. should be done now

**Review by Joehuu — APPROVED** [review]

**Pull request merged**